stock_scanner_web/views/web_templates.xml

```diff
@@ -0,0 +1,160 @@
+                                                </a>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </p>
+                            <p t-if="code in ['M','E']">
+
+                                <h3>
+                                    <span t-esc="result[0][1:]"/>
+                                </h3>
+                                <t t-foreach="result[1:]" t-as="line">
+                                    <p t-esc="line"/>
+                                </t>
+                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/0" class="btn">
+                                    Go
+                                </a>
+                            </p>
+                            <p t-if="code == 'F'">
+
+                                <h3>
+                                    <span t-esc="result[0][1:]"/>
+                                </h3>
+                                <t t-foreach="result[1:]" t-as="line">
+                                    <p t-esc="line"/>
+                                </t>
+                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/Menu" class="btn">
+                                    Go to menu
+                                </a>
+                            </p>
+                            <p t-if="code == 'C'">
+
+                                <h3>
+                                    <span t-esc="result[0][1:]"/>
+                                </h3>
+                                <t t-foreach="result[1:]" t-as="line">
+                                    <p t-esc="line"/>
+                                </t>
+                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/True" class="btn">
+                                    Yes
+                                </a>
+                                <a id="btn-go" t-attf-href="/scanner_call/{{terminal_number}}/action/False" class="btn">
+                                    No
+                                </a>
+                            </p>
+                            <p t-if="code in ['N','Q']">
+
+                                <h3>
+                                    <span t-esc="result[0][1:]"/>
+                                </h3>
+                                <t t-foreach="result[1:]" t-as="line">
+                                    <p t-esc="line"/>
+                                </t>
+
+                                <form method="get"
+                                    t-att-action="'/scanner_call/%s/action' % (terminal_number)" class="search_class form-inline">
+                                    <div class="input-group">
+                                        <input type="number" name="message" class="search-query form-control"/>
```

Comment:
The float step doesn't allow float values, only integers (tested on Firefox 49).